arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -421,7 +395,7 @@
         ],
         "responses": {
           "200": {
-            "description": "",
+            "description": "OK -- current usage acount and limit retrieved and returned successfully.",
```

Comment:
How about `OK -- current usage count and limit retrieved successfully.`

Typo in `acount`